### PR TITLE
Austin - Added helpmsg, updated main readme with UI

### PR DESCRIPTION
**Pull request by austin-currington**
I made a couple of related changes - I added a -help CLI flag which displays CLI flag usage and aborts. I also added some usage information on this and the GUI to the main readme.md file.

### CODE REVIEW

**Comment by austin-currington** [comment]
Oops, looks like I have some lingering issues with the cmdTest files. They're out of date now, because of auto-language-detection. I'll need to update them first.

**Comment by austin-currington** [comment]
I also want to add to the readme something about the windows context menu stuff, but I don't know how much detail to go into there.

File: CogNative/main.py [hunk 1]
```diff
@@ -17,6 +17,19 @@
 import sys
 args = sys.argv[1:]
 
+# -help flag displays help instead of processing anything
+if '-help' in args:
+    print("""
+CogNative CLI FLags: 
+    -sampleAudio <PATH>: audio file of voice to copy
```
Comment (andrew-fennell) [inline CogNative/main.py]:
```suggestion
    -sampleAudio <PATH>: audio file of voice to clone
```

Maybe this change to further distance ```-sampleAudio``` from ```-dialogueAudio```.

**Review by andrew-fennell — APPROVED** [review]
LGTM

**Pull request merged**